addons/l10n_latam_check/models/l10n_latam_check.py

```diff
@@ -35,9 +35,9 @@ class L10n_LatamCheck(models.Model):
         compute='_compute_current_journal', store=True,
     )
     name = fields.Char(string='Number')
-    bank_id = fields.Many2one(
-        comodel_name='res.bank',
-        compute='_compute_bank_id', store=True, readonly=False,
+    partner_bank_id = fields.Many2one(
```

Comment:
I'm not sure this makes sense

Reply:
Thanks @vbe-odoo

I was actually wondering whether we could remove it entirely 😈 ?
When a company receives a cheque, there generally isn't any need to know which bank issued it. You simply take it to the bank and deposit it into your account. 
For payments, one shouldn't need to know the account number nor bank of the recipient either.

(These are assumptions, having deposited cheques on behalf of others in my younger years, but never having received one myself 😀)